pandas/core/construction.py

```diff
@@ -582,28 +602,27 @@ def sanitize_array(
             # we will try to copy by-definition here
             subarr = _try_cast(data, dtype, copy)
 
-    elif isinstance(data, ABCExtensionArray):
-        # it is already ensured above this is not a PandasArray
-        subarr = data
-
-        if dtype is not None:
-            subarr = subarr.astype(dtype, copy=copy)
-        elif copy:
-            subarr = subarr.copy()
+    elif hasattr(data, "__array__"):
+        # e.g. dask array GH#38645
```

Comment:
@mroeschke one thought is here we could check for e.g. pyarrow array and avert a cast to ndarray.  most of the tests that get here do so with ChunkedArray and eventually cast back to ArrowArray

Reply:
So publicly this is the case when one calls e.g. `pd.Series(pa.chunked_array([[1, 2]]))`?

Reply:
right

Reply:
Okay cool. Will put up a PR in the next few days